libdokan/folderlist.go

```diff
@@ -86,7 +93,15 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 		}
 		fl.mu.Unlock()
 
+		h, er := libkbfs.ParseTlfHandle(
+			ctx, fl.fs.config.KBPKI(), name, fl.public)
+
 		if ok {
+			if er == nil {
+				if err = fl.addToFavorite(ctx, h); err != nil {
```

Comment:
Not sure why you need this call.  In this case, the node already exists in `fl.folders`, so shouldn't it have already been added to favorites?  If there's a good reason, please add a comment.